shared/api/instance.go

```diff
@@ -174,7 +174,7 @@ type InstancePut struct {
 // API extension: instances_rebuild.
 type InstanceRebuildPost struct {
 	// Rebuild source
-	Source InstanceSource `json:"source" yaml:"source"`
+	Source InstanceSource `yaml:",inline"`
```

Comment:
This is an api breakage I don't think we can change it.

What was the thinking behind this?

Reply:
It was just a test to see the output of the readthedoc documentation but it does not seem to be what I want. Now I reverted it.

Reply:
Now this https://canonical-ubuntu-documentation-library--12137.com.readthedocs.build/lxd/en/12137/api/#/instances/instance_rebuild_post looks good so I'd say this is ready to be merged. What do you think @ru-fu ?